tests/lsp/dir/in_dir.toit

```diff
@@ -1,6 +1,8 @@
-// Copyright (C) 2020 Toitware ApS.
-// Use of this source code is governed by a Zero-Clause BSD license that can
-// be found in the tests/LICENSE file.
+// The location of this toit file.
```

Comment:
What does this mean?

Reply:
Changed it to "// Keep this comment, as '@ in_dir' needs a line to point to.